templates/downloads/index.html

```diff
@@ -37,6 +37,59 @@ <h1 class="call-to-action">Download the latest version of Python</h1>
 {% endblock header_content %}
 
 {% block content %}
+                <div class="row active-release-list-widget">
+                    {% render_active_banner %}
```

Comment:
Is it possible to move this inside `<div class="row download-list-widget">`? If not, we need to remove the duplicate `render_active_banner` call in line 95.

Reply:
Actually, you can ignore my comment about moving it inside `<div class="row download-list-widget">`. It looks pretty great! :) We just need to remove the duplicate `render_active_banner` call in line 95.